xline/src/storage/lease_store/lease_collection.rs

```diff
@@ -0,0 +1,176 @@
+use std::{
+    collections::HashMap,
+    time::{Duration, Instant},
+};
+
+use clippy_utilities::Cast;
+use itertools::Itertools;
+use parking_lot::RwLock;
+use utils::parking_lot_lock::RwLockMap;
+
+use super::{lease_queue::LeaseQueue, Lease};
+use crate::{rpc::PbLease, storage::ExecuteError};
+
+/// Min lease ttl
+const MIN_LEASE_TTL: i64 = 1; // TODO: this num should calculated by election ticks and heartbeat
```

Comment:
Shall we calculate the `MIN_LEASE_TTL` by election ticks and heartbeat here in this pr?

Reply:
Added